<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SHORT_EN</class>
 <widget class="QWidget" name="SHORT_EN">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="back_button">
     <property name="text">
      <string>返回主菜单</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="srcFilePath_groupBox">
     <property name="title">
      <string>选择文件</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="srcFilePath_lineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="srcFilePath_button">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="dstFilePath_groupBox">
     <property name="title">
      <string>保存路径</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="dstFilePath_lineEdit"/>
      </item>
      <item>
       <widget class="QToolButton" name="dstFilePath_button">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mode_groupBox">
     <property name="title">
      <string>短块加密方式</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="mode1_radioButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>填充</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mode2_radioButton">
        <property name="text">
         <string>密文挪用</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mode3_radioButton">
        <property name="text">
         <string>序列密码</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>密钥(64个二进制位)</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLineEdit" name="key_lineEdit"/>
      </item>
      <item>
       <widget class="QToolButton" name="genKey_button">
        <property name="text">
         <string>随机生成</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="encrypt_button">
       <property name="text">
        <string>加密</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="decrypt_button">
       <property name="text">
        <string>解密</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
